extension.js

```diff
@@ -486,6 +488,17 @@ class Miniview {
         }
     }
 
+    _windowFocusMonitor(display) {
+        let activeWindow = display.get_focus_window();
+        //global.log(`miniview: _windowFocusMonitor   : display=${display} window=${activeWindow.get_title()}`);
+
+        if (this.lookupIndex(activeWindow) == this._winIdx) {
```

Comment:
I think you can get away with:
`if (activeWindow == this._metaWin) {`

Reply:
You are right, didn't notice the **_metaWin** property :-)